internal/mediainfo/language.go

```
@@ -1,140 +1,142 @@
-}
-
-func isDigit(s string) bool {
-	for i := 0; i < len(s); i++ {
-		c := s[i]
-		if c < '0' || c > '9' {
-			return false
-		}
-	}
-	return true
-}
+package mediainfo
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)